modules/store/spec/selector.spec.ts

```diff
@@ -126,13 +114,27 @@ describe('Selectors', () => {
       expect(projectFn).toHaveBeenCalledTimes(2);
     });
 
+    it('should not memoize last successful projection result in case of error', () => {
+      const firstState = { ok: 'a' };
+      const secondState = { faulty: 'b' };
+      const fail = () => {
+        throw new Error();
+      };
+      const selector = createSelector(
+        identity,
```

Comment:
Instead of using `identity`, let's just return state

```suggestion
        state => state,
```